src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,204 @@
+        List<Car> cars = new ArrayList<>();
+        String getCarsByDriverQuery = "SELECT c.id as car_id, model, manufacturer_id, "
+                + "m.name, m.country FROM cars c"
+                + " JOIN manufacturers m ON c.manufacturer_id = m.id"
+                + " JOIN cars_drivers cd ON c.id = cd.car_id WHERE cd.driver_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement getCarsStatement = connection
+                         .prepareStatement(getCarsByDriverQuery)) {
+            getCarsStatement.setLong(1, driverId);
+            ResultSet resultSet = getCarsStatement.executeQuery();
+            while (resultSet.next()) {
+                cars.add(getCarFromResultSet(resultSet));
+            }
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Couldn't get cars by driver id: "
+                    + driverId, throwable);
+        }
+        for (Car car : cars) {
+            car.setDrivers(getDriversForCar(car.getId()));
+        }
+        return cars;
+    }
+
+    private Car getCarFromResultSet(ResultSet resultSet) throws SQLException {
+        Car car = new Car();
+        car.setModel(resultSet.getString("model"));
+        car.setId(resultSet.getObject("car_id", Long.class));
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setId(resultSet.getObject("manufacturer_id", Long.class));
+        manufacturer.setName(resultSet.getString("name"));
+        manufacturer.setCountry(resultSet.getString("country"));
+        car.setManufacturer(manufacturer);
+        return car;
+    }
+
+    private List<Driver> getDriversForCar(Long id) {
+        List<Driver> drivers = new ArrayList<>();
+        String getDriversForCarQuery = "SELECT id, name, license_number FROM drivers d"
+                + " JOIN cars_drivers cd ON d.id = cd.driver_id WHERE cd.car_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement getDriversStatement = connection
+                         .prepareStatement(getDriversForCarQuery)) {
+            getDriversStatement.setLong(1, id);
+            ResultSet resultSet = getDriversStatement.executeQuery();
+            while (resultSet.next()) {
+                drivers.add(getDriverFromResultSet(resultSet));
+            }
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Can't get drivers for car with id " + id, throwable);
+        }
+        return drivers;
+    }
+
+    private Driver getDriverFromResultSet(ResultSet resultSet) throws SQLException {
+        Driver driver = new Driver();
+        driver.setId(resultSet.getObject("id", Long.class));
+        driver.setName(resultSet.getString("name"));
+        driver.setLicenseNumber(resultSet.getString("license_number"));
+        return driver;
+    }
+
+    private void insertRelationsForCar(Car car) {
```

Comment:
```suggestion
    private void insertDriversForCar(Car car) {
```

for now, it's only one relation